benchmark/index.js

```diff
@@ -298,25 +286,101 @@ document.addEventListener('DOMContentLoaded', () => {
     modelName: 'mobilenet',
     iteration: 0
   }];
-  let webmlAPIConfigurations = [{
+  let webmlAPIConfigurationsAndroid = [{
+    framework: 'Web ML API',
+    backend: 'NN',
```

Comment:
How about let user to choose preference? I may not expose the underlying backend here.